weave-js/src/core/client/cachePolicy.ts

```diff
@@ -24,12 +24,33 @@ const getStringValFromNode = (
   return undefined;
 };
 
+const WANDB_COMMIT_HASH_LENGTH = 20;
+
+const isHexString = (s: string): boolean => {
+  return s.length % 2 === 0 && /^[0-9a-f]+$/i.test(s);
+};
+
+const likelyCommitHash = (version: string): boolean => {
```

Comment:
Maybe continue with the `is` prefixed naming for a boolean result?

Reply:
👍